byzcoin/service.go

```diff
@@ -2163,7 +2173,7 @@ func (s *Service) monitorLeaderFailure() {
 // registerContract stores the contract in a map and will
```

Comment:
I think this is not called anymore?

Reply:
I kept it for backward compatible reasons, but I am happy to remove it.

Reply:
Actually we can't just removing it because it is used in other places.